### PR TITLE
Add starrocks plugin

### PR DESCRIPTION
**Pull request by long2ice**
# Summary

This plugin is used to migrate collected data from database(MySQL/PostgreSQL) to starrocks
### Does this close any open issues?
Please mention the issues here.

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: config-ui/src/data/pipeline-config-samples/starrocks.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        be_port: 8040,
+        tables: ['_tool_github_commits']
+      }
+    }
+  ]
+]
+
+export {
+  starRocksConfig
+}
```
Comment (Singh212dab) [inline config-ui/src/data/pipeline-config-samples/starrocks.js:20]: };

**Review by Singh212dab — CHANGES_REQUESTED** [review]

File: config-ui/src/data/Providers.js [hunk 2]
```diff
@@ -37,7 +37,8 @@ const Providers = {
   GITEXTRACTOR: 'gitextractor',
   FEISHU: 'feishu',
   AE: 'ae',
-  DBT: 'dbt'
+  DBT: 'dbt',
+  STARROCKS: 'starrocks',
 }
```
Comment (Singh212dab) [inline config-ui/src/data/Providers.js:42]: };

**Review by Singh212dab — CHANGES_REQUESTED** [review]

File: plugins/starrocks/tasks.go [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+	if err != nil {
+		panic(err)
+	}
+	client := http.Client{}
+	req, err := http.NewRequest(http.MethodPut, url, bytes.NewBuffer(jsonData))
+	if err != nil {
+		panic(err)
+	}
+	req.SetBasicAuth(config.User, config.Password)
+	for k, v := range headers {
```
Comment (Singh212dab) [inline plugins/starrocks/tasks.go:89]: for (k, v := range headers)

**Review by Singh212dab — CHANGES_REQUESTED** [review]

File: config-ui/src/data/Providers.js (hunk 2 above)
Comment (warren830) [reply config-ui/src/data/Providers.js:42]: > };

Thanks for your comment! Would you like to join us on slack, then we can discuss some problems together

**Comment by thenicetgp** [comment]
@long2ice Hi, thank you for your contribution to Apache Devlake, Devlake community specially invites you to join our contributor group and issue you a contributor certificate. If you are willing, please contact 13120413800@163.com, looking forward to your reply.

**Pull request merged**